core/ui/LayoutSwitcher/Layout.tid

```diff
@@ -0,0 +1,20 @@
+title: $:/core/ui/LayoutSwitcher/Layout
+tags: $:/tags/LayoutSwitcher $:/tags/ControlPanel/Appearance
+caption: {{$:/language/LayoutSwitcher/Hint}}
+
+<$linkcatcher to="$:/layout">
+<div class="tc-chooser">
+<$list filter="[all[tiddlers+shadows]tag[$:/tags/Layout]] [[$:/core/ui/PageTemplate]] +[sort[name]]">
+<$list filter="[{$:/layout}!has[text]]" variable="ignore" emptyMessage="""
+<$set name="cls" filter="[all[current]field:title{$:/layout}]" value="tc-chooser-item tc-chosen" emptyValue="tc-chooser-item"><div class=<<cls>>><$link to={{!!title}}>''<$transclude field="name"/>'' - <$transclude field="description"/></$link></div>
+</$set>
+""">
+<$set name="cls" filter="[all[current]field:title[$:/core/ui/PageTemplate]]" value="tc-chooser-item tc-chosen" emptyValue="tc-chooser-item"><div class=<<cls>>><$link to={{!!title}}>''<$transclude field="name"/>'' - <$transclude field="description"/></$link></div>
+</$set>
+</$list>
+</$list>
+</div>
+</$linkcatcher>
+
+<br>
+<$button>{{$:/language/RootTemplate/Reset}}<$action-deletetiddler $tiddler="$:/core/ui/RootTemplate"/></$button>
```

Comment:
What if we only display this button when the tiddler $:/core/ui/RootTemplate exists as a non-shadow tiddler?
And maybe call it "Reset to layout defaults".

Reply:
That makes sense! I'll add that check

Reply:
I'm not convinced that we need this reset feature. The UI complicates things, and it's hard to explain satisfactorily to end users because it only makes sense for quite sophisticated users.

Reply:
Ok @Jermolene, I'll remove this button